packages/grafana-schema/src/schema/mudball.gen.ts

```diff
@@ -393,14 +393,21 @@ export type FieldTextAlignment = ('auto' | 'left' | 'right' | 'center');
  */
 export enum TableCellDisplayMode {
   Auto = 'auto',
-  BasicGauge = 'basic',
   ColorBackground = 'color-background',
+  // Deprecated
   ColorBackgroundSolid = 'color-background-solid',
   ColorText = 'color-text',
-  GradientGauge = 'gradient-gauge',
+  Gauge = 'gauge',
   Image = 'image',
   JSONView = 'json-view',
-  LcdGauge = 'lcd-gauge',
+}
+
+/**
+ * TODO Docs
+ */
+export enum BackgroundDisplayMode {
```

Comment:
```suggestion
export enum TableColoredCellBackgroundDisplayMode {
```

Remember these are exported from a single namespace

Reply:
Updated the name of enum 😄